website/static/js/osfHelpers.js

```diff
@@ -590,14 +590,18 @@ function getNodesOriginal(nodeTree, nodesOriginal) {
         }
     }
     nodesOriginal[nodeId] = {
-        public: nodeTree.node.is_public,
+        isPublic: nodeTree.node.is_public,
```

Comment:
This `getNodesOriginal` should probably go in `projectSettingsTreebeardBase` along with the other shared utility functions (see https://github.com/CenterForOpenScience/osf.io/pull/4958/files#r55267144).